ui-components/typesoftoolbarbuttons.md

```diff
@@ -97,20 +97,18 @@ The callback given to `editor.formatter.formatChanged` is a function that takes
+|--------------|-----------------------------------| ----------------- |-------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| text         | string                            | Primary button    | optional    | Text displayed if no icon is found.                                                                                                                                                                                         |
+| icon         | string                            | Primary button    | optional    | Name of the icon to be displayed. Must correspond to an icon in the [icon pack]({{site.baseurl}}/advanced/editor-icon-identifiers/) or an icon in a [custom icon pack]({{site.baseurl}}/advanced/creating-an-icon-pack/). |
+| onAction     | (api) => void                     | Primary button    | required    | Function invoked when the basic button section is clicked.                                                                                                                                                                  |
+| select       | (value: string) => boolean        | Primary button    | optional    | default: false - Function run on each option to determine if it should be highlighted as active.                                                                                                                            |
+| columns      | string                            | Drop-down menu    | optional    | default: 1 - Number of columns for the list of options.                                                                                                                                                                     |
+| fetch        | (success: (menu) => void) => void | Drop-down menu    | required    | Function that takes a callback which must be passed the list of options for the button's dropdown.                                                                                                                          |
+| onItemAction | (api, value) => void              | Choice menu items | required    | Function invoked when a dropdown list option is clicked.                                                                                                                                                                    |
+| onSetup      | (api) => (api) => void            | Choice menu items | optional    | default: () => () => {} - Function invoked when the button is rendered.                                                                                                                                                     |
```

Comment:
This is incorrect for the target, as it applies to the entire component.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1573/commits/d6f9cf4a8078de019a983938369aca2dfef00e0b